src/lxc/monitor.c

```diff
@@ -181,7 +181,6 @@ int lxc_monitor_sock_name(const char *lxcpath, struct sockaddr_un *addr) {
 	if (ret < 0)
 		return -1;
 
-	sockname[sizeof(addr->sun_path)-3] = '\0';
```

Comment:
`(sockname = sun_path+1) + sizeof(sun_path) - 3 = sun_path + sizeof(sun_path) - 2`
So this would set `sun_path[sizeof(sun_path)-2] = 0` which is the second to last byte of `sun_path` (while IMO only the last one should need to be `\0`? (and first one sure, but this is about the size of `sun_path`, not the length of its contents so the initial byte's content doesn't matter for this offset)).